### DISCUSSION TITLE
卡齐娜真可爱

### DISCUSSION THREAD

**Discussion by Dingdong1018** [share121/inter-knot#5150] [原神]
![IMG_20240913_172627](https://github.com/user-attachments/assets/0f897e9c-b4ca-420f-894b-ec0f7038a4d1)

**Comment by faatong** [comment]
可爱<img src="https://bbs-static.miyoushe.com/static/2024/09/04/70420b7df8a3934cb8e7b0f0f4136511_7657550741734618773.png" alt="卡齐娜-囤货" width="75" >

**Comment by keyuanya** [comment]
幻梦的壁纸
![c648ffd92573c1bbe143b84ca6b75abd7efa4a6d7965c6fb3a75761e0439e00f_upscayl_3x_remacri](https://github.com/user-attachments/assets/a46a3070-a479-4ab0-bd5a-4a6b7a8d14a3)

**Reply by Dingdong1018** [reply]
本来想用另一张的（
